deploy/kubernetes/base/controller.yaml

```diff
@@ -62,6 +62,23 @@ spec:
             timeoutSeconds: 3
             periodSeconds: 10
             failureThreshold: 5
+        - name: node-update-controller
+          image: quay.io/powercloud/ibm-powervs-block-csi-driver:v0.0.1
```

Comment:
somehow we need to get rid of this image `quay.io/powercloud/ibm-powervs-block-csi-driver:v0.0.1` of course not via this PR but we need to remove from everywhere.

Reply:
Yes, we can give gcr path: **gcr.io/k8s-staging-cloud-provider-ibm/ibm-powervs-block-csi-driver:v1.0.0**

Reply:
how about `gcr.io/k8s-staging-cloud-provider-ibm/ibm-powervs-block-csi-driver:dev` something more meaningful and assume this will get overridden with an actual image later.

Reply:
Will be changing all the image Paths to **gcr.io/k8s-staging-cloud-provider-ibm/ibm-powervs-block-csi-driver:dev** in a new PR.